collections/ansible_collections/cloudkit/service/roles/cleanup_stale_network_resources/meta/argument_specs.yaml

```diff
@@ -0,0 +1,11 @@
+argument_specs:
+  main:
+    options:
+      cleanup_stale_network_resources_osac_tag:
+        type: str
+        required: true
+      cleanup_stale_network_resources_instance_name:
```

Comment:
I think this should be called `cleanup_stale_network_resources_namespace`, since we're explicitly using it for that purpose in e.g. the "Get all existing ClusterOrders" task.